services/blockchain-indexer/shared/dataService/business/token.js

```diff
@@ -86,30 +84,52 @@ const getTokenTopBalances = async (params) => {
 		data: {},
 		meta: {},
 	};
+
+	const { search, ...remParams } = params;
```

Comment:
We need to remove search from params or else it will be passed on to the database [here](https://github.com/LiskHQ/lisk-service/blob/1695-extend-the-top-balances-endpoint/services/blockchain-indexer/shared/dataService/business/token.js#L111) which will provide inconsistent results.